desc/compute/_omnigenity.py

```diff
@@ -346,6 +348,138 @@ def _f_T(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="eta",
+    label="\\eta",
+    units="rad",
+    units_long="radians",
+    description="Intermediate omnigenity coordinate along field lines",
+    dim=1,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="rtz",
+    data=[],
+    parameterization="desc.magnetic_fields.OmnigenousField",
+)
+def _eta(params, transforms, profiles, data, **kwargs):
+    # theta is used as a placeholder for eta (angle along field lines)
+    data["eta"] = (transforms["grid"].nodes[:, 1] - jnp.pi) / 2
+    return data
+
+
+@register_compute_fun(
+    name="alpha",
+    label="\\alpha",
+    units="rad",
+    units_long="radians",
+    description="Field line label, defined on [0, 2pi)",
+    dim=1,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="rtz",
+    data=["NFP"],
+    parameterization="desc.magnetic_fields.OmnigenousField",
+)
+def _alpha(params, transforms, profiles, data, **kwargs):
+    # zeta is used as a placeholder for alpha (field line label)
+    data["alpha"] = transforms["grid"].nodes[:, 2] * data["NFP"]
+    return data
+
+
+@register_compute_fun(
+    name="h",
+    label="h = \\theta + (N / M) \\zeta",
+    units="rad",
+    units_long="radians",
+    description="Omnigenity symmetry angle",
+    dim=1,
+    params=["x_lmn"],
+    transforms={"x": [[0, 0, 0]]},
+    profiles=[],
+    coordinates="rtz",
+    data=["rho", "alpha", "eta"],
+    parameterization="desc.magnetic_fields.OmnigenousField",
+)
+def _omni_angle(params, transforms, profiles, data, **kwargs):
+    nodes = jnp.array([data["rho"], data["eta"], data["alpha"]]).T
+    data["h"] = (
+        transforms["x"].basis.evaluate(nodes) @ params["x_lmn"]
```

Comment:
aren't the nodes constant here? why can't we precompute the transform?

Reply:
precompute in build, use `Grid` class maybe with `sort=False` `jitable=True`